migrations/2023-10-17-084554_alter_payout_identifier_length/up.sql

```diff
@@ -0,0 +1,24 @@
+-- Payout Attempt table
+ALTER TABLE
```

Comment:
Alter table will result in DB downtime. So requests might fail during the migration. 
Please check with @venkatesh-juspay on how do we go with this. 
Worst case solution would be creating new column with the new size. New column is not a problem as it won't cause downtime.

Reply:
Removed this change.

`payout_attempt_id` will be created using payout_id and attempt_count now.

![image](https://github.com/juspay/hyperswitch/assets/46213975/c31f5133-312f-4fee-9703-c36d21b01103)